### PR TITLE
Project overview API

### PR DESCRIPTION
**Pull request by aksharahegde**

### CODE REVIEW

**Comment by fayazara** [comment]
@aksharahegde Is this url correct?

`/api/projects/overview?start=12-12-2020&end=12-12-2021`

What format should I send the dates in?

**Comment by aksharahegde** [comment]
/api/projects/{projectId}/overview

no date is needed

File: server/api/projects/[id]/overview.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  /*
+    Daily -> Hourly stats
+    Weekly -> Daywise stats
+    Monthly -> Weekly stats
+    Yearly -> Monthly stats
+  */
+
+  const { getProjectId, getDateRangeFilter } = useValidation(event);
+  const projectId = await getProjectId();
+  const { start, end } = await getDateRangeFilter();
```
Comment (fayazara) [inline server/api/projects/[id]/overview.ts]: What do these start and end variables do?

Comment (aksharahegde) [reply server/api/projects/[id]/overview.ts:37]: uh. i forgot to remove. :-) Will update the PR

Comment (aksharahegde) [reply server/api/projects/[id]/overview.ts]: it should be in ISO format.
```
new Date().toISOString()
'2023-08-09T13:54:45.021Z'
```

**Pull request merged**